src/main/java/org/apache/pirk/query/wideskies/Query.java

```diff
@@ -136,89 +124,30 @@ public boolean containsElement(BigInteger element)
     return queryElements.containsValue(element);
   }
 
-  public void clearElements()
-  {
-    queryElements.clear();
-  }
-
   /**
    * This should be called after all query elements have been added in order to generate the expTable. For int exponentiation with BigIntegers, assumes that
    * dataPartitionBitSize < 32.
-   *
    */
-  public void generateExpTable(int numThreads) throws InterruptedException
+  public void generateExpTable()
```

Comment:
I want to draw attention to this part of the new code, that will use the ```ForkJoinPool#commonPool()``` (i.e. all available processors by default).  We can put this into a smaller custom pool if it is desirable to have that restriction.